docs/resources/cc_connection.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: Cloud Connect (CC)
+---
+
+# huaweicloud_cc_connection
+
+Manages a Cloud Connection resource within HuaweiCloud.  
+Cloud Connect (CC) is a service that enables you to quickly build ultra-fast, high-quality, and stable networks
+between VPCs across regions and between VPCs and on-premises data centers.
```

Comment:
duplicate `between VPCs`?

Reply:
This sentence comes from CC's official documentation.  https://support.huaweicloud.com/intl/en-us/cc/
so keep it